client/html/layouts/basket/product/body-default.html

```diff
@@ -0,0 +1,114 @@
+<?php
```

Comment:
Most likely you have a mistake in the template file names in  client/html/src/Client/Html/Basket/Mini/Product/Default.php. You can see more details for the problem in the madmin_log table.